themes/paper/assets/main.css

```diff
@@ -904,13 +904,13 @@ body {
   padding-left: 1em;
 }
 
-.prose :where(blockquote p:first-of-type):not(:where([class~="not-prose"],[class~="not-prose"] *))::before {
+/* .prose :where(blockquote p:first-of-type):not(:where([class~="not-prose"],[class~="not-prose"] *))::before {
   content: open-quote;
 }
 
 .prose :where(blockquote p:last-of-type):not(:where([class~="not-prose"],[class~="not-prose"] *))::after {
   content: close-quote;
-}
+} */
```

Comment:
It appears impossible to attribute a pull quote in correctly without hiding the auto quote marks? I think quotes without attribution, and without quote marks, still look like quotes so _I_ don't think this breaks old posts.